code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -578,6 +578,27 @@
 	tastes = list("cookie" = 1)
 	foodtype = GRAIN | SUGAR
 	dunkable = TRUE
+	obj_flags = NONE // you don't cheat cookie's name. this removes `UNIQUE_RENAME` flag.
```

Comment:
I don't understand this comment. what does UNIQUE_RENAME do

and why

Reply:
That allows you to change the item's name and desc into what you'd like. I don't know why this flag is turned on to all food types.

Reply:
If someone wants to name the cookie holy cookie they can... no need to remove the flag